apps/src/componentLibrary/tags/tags.module.scss

```diff
@@ -0,0 +1,113 @@
+@import "color.scss";
+@import "font.scss";
+@import "@cdo/apps/componentLibrary/typography/typography.module";
+
+// Tags common styles
+.tags {
+  display: flex;
+  column-gap: 0.5em;
+
+  > div {
+    display: flex;
+    align-items: baseline;
+    gap: 2px;
+    color: $neutral-dark;
+    padding: 2px 12px;
+    background-color: $brand_primary_light;
+    border-radius: 6.25rem;;
+    max-width: 15em;
+
+    span {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+
+    i {
+      display: flex;
+      justify-content: center;
+    }
+
+    &:has(i:first-child) {
+      padding-inline-start: 8px;
+    }
+
+    &:has(i:not(:first-child)) {
+      padding-inline-end: 8px;
+    }
+  }
+}
+
+.tagTooltip {
+  opacity: 1;
+  margin-bottom: 0.75em;
+  pointer-events: none;
+
+  :global(.tooltip-inner) { /* stylelint-disable-line selector-pseudo-class-no-unknown */
+    border-radius: 5px;
+    color: #fff;
+    background-color: #222;
```

Comment:
Should we be using our own color constants?

Reply:
oh yeah, that's the copy-pasted tooltip styles from Meg's CardLabels, we'll have our own tooltips in scope of DSCO Phase 2, so this was a blind spot to me, thanks for noticing!

I believe yes, and even further - it'll be better to use DSCO tooltip color from the start. (`#222` is not even a part of our DSCO colors)